Adding undo search

mohan440 opened the pull request.

mandarini commented:
Hi Mohan, in general, looks good. I will review in more detail, but just a comment for now, I would suggest you try to trigger the snackbar from within the ngrx effects. Also, please try to squash your commits and only have one commit in the end! :)

mohan440 commented:
> Hi Mohan, in general, looks good. I will review in more detail, but just a comment for now, I would suggest you try to trigger the snackbar from within the ngrx effects. Also, please try to squash your commits and only have one commit in the end! :)

Added the logic to ngrx effects and squashed the commits

mandarini reviewed (approved):
Looks good to me! You can move on to task 4!
